scripts/partition_manager/partition_manager.rst

```diff
@@ -215,6 +214,30 @@ share_size: list
    If none of the partitions in the ``share_size`` list exists, and the partition does not define a ``size`` property, then the partition is removed.
    If none of the partitions in the ``share_size`` list exists, and the partition **does** define a ``size`` property, then the ``size`` property is used to set the size.
 
+All occurrences of a partition name can be replaced with a dict with the key ``one_of``, which is resolved to the first existing partition in the ``one_of`` value.
+It is not allowed to have the ``app`` partition inside such a ``one_of`` dict, because the ``app`` partition always exists.
```

Comment:
Why do we have this restriction? Putting 'app' as the last entry in the list seems to me like a valid use case.

Reply:
You are correct, I added a test with 'app' inside the 'one_of' and will remove this line.